src/target/riscv/riscv.h

```diff
@@ -129,6 +129,22 @@ struct reg_name_table {
 	char **reg_names;
 };
 
+typedef struct riscv_mem_access_info {
+	target_addr_t target_address;
```

Comment:
Since there is no other address in this structure, could it be called just `address` for simplicity?

One might also argue that in case of read operations, it is actually a `source_address`. :)

```suggestion
	target_addr_t address;
```

Reply:
Addressed